tests/decode/test5_corner_cases/src/main.c

```diff
@@ -179,6 +199,35 @@ void test_numbers2(void)
 
 	zassert_equal(ZCBOR_ERR_WRONG_TYPE, cbor_decode_Numbers2(payload_numbers2_inv6,
 		sizeof(payload_numbers2_inv6), &numbers2, &decode_len), NULL);
+
+	zassert_equal(ZCBOR_ERR_WRONG_VALUE, cbor_decode_Numbers2(payload_numbers2_inv7,
+		sizeof(payload_numbers2_inv7), &numbers2, &decode_len), NULL);
+}
+
+void test_tagged_union(void)
+{
+	size_t decode_len;
+	const uint8_t payload_tagged_union1[] = {0xD9, 0x10, 0xE1, 0xF5};
+	const uint8_t payload_tagged_union2[] = {0xD9, 0x09, 0x29, 0x10};
+	const uint8_t payload_tagged_union3_inv[] = {0xD9, 0x10, 0xE1, 0x10};
```

Comment:
We have comments for most other types in the tests describing what type it is. Should we add that here as well? I see the variable name, but to follow convention and then we can have the data more readily available.

Reply:
Good idea. Added a sentence.